components/layouts/Layout.tsx

```diff
@@ -75,9 +77,9 @@ const Layout: React.FunctionComponent<Props> = props => {
 
             <NavBar />
 
-            <div className="container">
+            <Container>
```

Comment:
This may be the container style, or it may be a column style, but I noticed that on my mobile (Android / Chrome browser) that it is rendering as a little wider than the browser screen so it can be moved around. 

Here is the prod learn site:

![Screenshot_20210809-155956](https://user-images.githubusercontent.com/5977693/128767478-713880bf-5e13-4657-b95f-653ae378541e.png)

Here is the deploy preview on load:

![Screenshot_20210809-160002](https://user-images.githubusercontent.com/5977693/128767520-ff58534a-c59e-46ba-b2cc-0fbd7877165c.png)

Showing horizontal scroll of deploy preview:

![Screenshot_20210809-160009](https://user-images.githubusercontent.com/5977693/128767570-95983e50-311a-48f2-b616-755478634c7b.png)

Reply:
This should be resolved now with some more queries/breakpoints on `Container` and `Footer`

Reply:
I'm still having the viewport issue on Android, are you not getting it on iPhone? Text is a bit cut off as are the images on the right side of the window for me.

![Screenshot_20210809-175355](https://user-images.githubusercontent.com/5977693/128779516-92eddb0e-ef75-4802-b271-3d62dd4cf2a5.png)

Reply:
It's a little hard to tell since you don't see the border in the image, for example the line under Step 1 has the "r" in "user" not fully displayed in the viewport, compare to the other "r"s.

Reply:
I think this last commit captures it! Using `576px` as default across `Footer`, `Row`, and `Container`

Reply:
Yes, great!